### DISCUSSION TITLE
Is multiple displays supported?

### DISCUSSION THREAD

**Discussion by JesperMell** [DeviceFarmer/stf#574] [Q&A]
Greetings!

I have a device that uses two physical displays and I usually use scrcpy (https://github.com/Genymobile/scrcpy) for viewing both screens. Does stf support displaying multiple, or just 2, displays?

**Comment by pcrepieux** [comment]
This is currently not supported by STF but minicap (which is the underlying screen streaming utility)  is supposed to handle multiple screens. At least, one can request for a specific display id. I confess I have no idea at the moment what would be the effort required to integrate this in STF. Any thoughts welcome.
